shared/terraform.go

```diff
@@ -38,18 +49,26 @@ func loadTFconfig(
 	t *testing.T,
 	varDir string,
 	terraformOptions *terraform.Options,
-	product string,
 ) (*Cluster, error) {
 	c := &Cluster{}
+	cfg, err := config.AddEnv()
+	if err != nil {
+		return nil, err
+	}
 
-	loadTFoutput(t, terraformOptions, c)
+	loadTFoutput(t, terraformOptions, c, cfg.Module)
 	loadAwsEc2(t, varDir, c)
-	if product == "rke2" {
+	if cfg.Product == "rke2" {
 		loadWinTFCfg(t, varDir, terraformOptions, c)
 	}
 
 	c.Config.Arch = terraform.GetVariableAsStringFromVarFile(t, varDir, "arch")
-	c.Config.Product = product
+	if c.Config.Arch == "arm" {
+		c.Config.Arch = "arm64"
```

Comment:
this will break right? since our folder is arm not arm64

Reply:
Yes will update the script to handle it

Reply:
Updated, the script already handles it :)